src/browser/components/molecules/OverdueTaskList/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import {fetchURL, toStandardDate } from '../../../common/utils'
+import moment from 'moment'
+import styles from './index.css'
+
+export default class OverdueTaskList extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      tasks:[]
+    }
+  }
+
+  componentDidMount(){
+    fetchURL('api/task/tasksAndUsers')
+    .then(results => {
+    this.setState({
+      tasks: results.filter(this.isOverdue),
+      })
+    })
+    .catch(err => {
+      return err
+    })
+  }
+
+
+  findEmailByUUID( user_id ){
```

Comment:
why does this function exist? Since you are now fetching the task, and the associated user of the task(by joining the task and user table), the email should be a property on the task object.